internal/outputgenerators/bonnmotion.go

```diff
@@ -156,27 +164,43 @@ func (b Bonnmotion) Generate(exp experiment.Experiment) {
 	}
 	b.outputFolder = outputFolder
 
-	b.stepFilePath = filepath.Join(b.outputFolder, BonnMotionStepFile)
-	allowedToWriteStepFile := folderstructure.MayCreatePath(b.stepFilePath)
+	stepFilePath := filepath.Join(b.outputFolder, BonnMotionStepFile)
+	allowedToWriteStepFile := folderstructure.MayCreatePath(stepFilePath)
 	if !allowedToWriteStepFile {
 		logger.Error("Not allowed to write step file!")
 		return
 	}
+	b.stepFile, err = os.OpenFile(stepFilePath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		logger.Error("Error opening step file:", err)
+		return
```

Comment:
please stay consistent with return and return nil

Reply:
In fact, the code is consistent. `return nil` is used for functions that return an `error` as only return value, `return` without `nil`) is used for functions, that don't return anything